src/js/landing.js

```
@@ -0,0 +1,412 @@
+
+  steps.forEach((step) => observer.observe(step));
+}
+
+// ── Live Statistics ───────────────────────────────────────────────────────────
+
+/**
+ * Calculates and renders live org statistics from the ORGS dataset.
+ * No-ops gracefully if ORGS is not available.
+ */
+function renderLiveStats() {
+  if (typeof ORGS === 'undefined' || !Array.isArray(ORGS)) { return; }
```

ESLint

[error] 'ORGS' is not defined.

(no-undef)